src/routes/living-manage-offer/containers/LivingManageOfferPage.js

```diff
@@ -30,14 +29,25 @@ export class LivingManageOfferPage extends React.Component {
       body: body && JSON.stringify(body)
     })
       .then(response => {
-        if (response.status === STATUS_OK) {
-          this.setState({success: response.status === STATUS_OK, sending: false})
+        if (response.status === OK) {
+          this.setState({success: response.status === OK, sending: false, serverError: null})
         } else {
-          response.text().then(text => this.setState({success: false, sending: false, serverError: text}))
+          response.json().then(error => this.setState({
+            success: false,
+            sending: false,
+            serverError: {
+              status: response.status,
+              message: error.errorMessage
+            }
+          }))
         }
       })
       .catch(() => {
-        this.setState({success: false, sending: false, serverError: 'Verbindung fehlgeschlagen'})
+        this.setState({
+          success: false,
+          sending: false,
+          serverError: {status: 0, message: 'Die Verbindung ist fehlgeschlagen.'}
```

Comment:
Hardcoded message, but stuff like this is not important here right?

Reply:
Hmm.. actually it is important... I haven't payed attention to that yet -.- I'll create a task to refactor hardcoded text 👍